bedrock/legal/urls.py

```diff
@@ -10,7 +10,10 @@
 from bedrock.legal_docs.views import LegalDocView
 
 urlpatterns = (
-    page('', 'legal/index.html'),
+    # page('', 'legal/index.html'),
+    url(r'^$',
+        views.legal_newsletter_test,
+        name='legal.index'),
```

Comment:
Even if it's coming back later, should remove the commented out line. Could also put the `url` call/args on a single line for conformity's sake.